src/components/pages/FAQPage/styles.tsx

```diff
@@ -8,12 +8,12 @@ export const StyleFAQSection = styled.div`
     flex-direction: column;
     justify-content: space-between;
     margin: ${space.m} auto;
-    width: 90vw;
     
 
     ${media.med`
-        height: 40vh;
+        height: 70vh;
```

Comment:
Same comment. Why are you defining the `height`? 

Working with heights in CSS is a pain in the ass, as a rule of thumb, you need to avoid giving a specific height to a component.

Reply:
Is it better to avoid height/width altogether and stick to max/min height/width for CSS (and define height/width in Gatsby)? 

What I was trying to do here was make sure the FAQ items displayed horizontally once the display size was large enough (as opposed to the top-down listing for mobile). Can max/min height/width handle that more flexibly/safely?

Reply:
>Is it better to avoid height/width altogether and stick to max/min height/width for CSS (and define height/width in Gatsby)?

Depends on the case. But, for `responsive design` I found that is easier to define `max/min height/width` But. sometimes you need to use  `max/min height/width` in combination with width.

For example:
```
width: 12vw;
max-width: 300px;
```

> What I was trying to do here was make sure the FAQ items displayed horizontally once the display size was large enough (as opposed to the top-down listing for mobile). Can max/min height/width handle that more flexibly/safely?

I didn't understand but leave it as is for now😄. Once I pull the code I'll check it out again.

Since you're building an accordion, you might be interested in this talk
https://www.youtube.com/watch?v=DNGGzwmfouU

Here is the repo:
https://github.com/joshwcomeau/talk-2019/tree/master/src/components/Accordion

Reply:
That was an interesting talk. I hadn't thought much about the performance side, especially for animation. 

As for CSS, I'll try to stick with max/min height/width for now.